### PR TITLE
Cumulative() function has incorrect doc string

### PR DESCRIPTION
**Pull request by cbowman0**
* Change doc string to be correct
* Add test for cumulative()
* Add test for consolidateBy()

### CODE REVIEW

File: webapp/graphite/render/functions.py [hunk 1]
```diff
@@ -983,11 +983,9 @@ def movingAverage(requestContext, seriesList, windowSize):
 
   return result
 
-def cumulative(requestContext, seriesList, consolidationFunc='sum'):
+def cumulative(requestContext, seriesList):
   """
-  Takes one metric or a wildcard seriesList, and an optional function.
-
-  Valid functions are 'sum', 'average', 'min', and 'max'
+  Takes one metric or a wildcard seriesList
```
Comment (obfuscurity) [inline webapp/graphite/render/functions.py]: Add a period here?

File: webapp/graphite/render/functions.py [hunk 2]
```diff
@@ -983,11 +983,9 @@ def movingAverage(requestContext, seriesList, windowSize):
...
 
   Sets the consolidation function to 'sum' for the given metric seriesList.
```
Comment (obfuscurity) [inline webapp/graphite/render/functions.py]: Do you think we should expand on this for people who might not read ahead to what `consolidateBy` does?

File: webapp/graphite/render/functions.py (hunk 1 above)
Comment (cbowman0) [inline webapp/graphite/render/functions.py]: Sure.

File: webapp/graphite/render/functions.py (hunk 2 above)
Comment (cbowman0) [inline webapp/graphite/render/functions.py]: I can copy in some of the verbiage about what consolidateBy does.  That would make it cleaner.

Comment (obfuscurity) [inline webapp/graphite/render/functions.py]: 💯

**Comment by cbowman0** [comment]
OK now?

**Comment by obfuscurity** [comment]
Thanks!

**Pull request merged**